RevenueCat/Scripts/Purchases.cs

```diff
@@ -18,7 +18,8 @@ public partial class Purchases : MonoBehaviour
     public delegate void GetOfferingsFunc(Offerings offerings, Error error);
 
     public delegate void CheckTrialOrIntroductoryPriceEligibilityFunc(Dictionary<string, IntroEligibility> products);
-
+    
+    public delegate void CanMakePaymentsFunc(bool canMakePayments, Error error);
```

Comment:
I know we don't have docs for the other functions, but I think we should start adding them (and add for all the other functions on a Friday). Do you mind adding docs here?

Reply:
@vegaro created clubhouse ticket for the others: https://app.clubhouse.io/revenuecat/story/7152/add-documentation-to-unity-functions, as well as a ticket for researching/setting up unit testing (if you could add comments on that tool you saw, that'd be great...): https://app.clubhouse.io/revenuecat/story/7153/research-set-up-unity-unit-tests